test/evergreen.yml

```diff
@@ -1533,6 +1533,15 @@ tasks:
         vars:
           test_name: wt8659_reconstruct_database_from_logs
 
+  - name: csuite-wt8963-insert-stress-test
```

Comment:
This definition isn't triggering any evergreen tests. To do so it needs to be called directly by another task or have one of its tags called (e.g. `"- name: ".pull_request"` in one of the buildvariants).

We can remove this section. All csuite tests are automatically executed by `make-check-test` which I've added to our latest patch build to confirm we're executing this test

Reply:
Fixed e1881302af50dfe77e6002e38f4a4367537a436d